modules/core/src/main/java/org/apache/ignite/internal/processors/cache/verify/IdleVerifyUtility.java

```diff
@@ -109,6 +112,70 @@ public static boolean isCheckpointNow(@Nullable IgniteCacheDatabaseSharedManager
         return progress != null && progress.inProgress();
     }
 
+    /**
+     * Gather updateCounters info.
+     *
+     * @param ign Ignite instance.
+     * @param grpIds Group Id`s.
+     * @return Counters info.
+     */
+    public static T2<Set<Integer>, Map<Integer, Set<Map.Entry<Integer, Long>>>> updCountersSnapshot(
```

Comment:
Consider using small inner class instead of T2. This will allow you to use descriptive methods names. Or extend documentation for return type